plugins/plugin-chart-word-cloud/src/chart/WordCloud.tsx

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import cloudLayout, { Word } from 'd3-cloud';
 import { PlainObject, createEncoderFactory, DeriveEncoding } from 'encodable';
 import { SupersetThemeProps, withTheme } from '@superset-ui/core';
+import configureEncodable from '../configureEncodable';
+
+configureEncodable();
```

Comment:
@ktmud I wasn't sure where/how to call this "the right way", any pointers would be appreciated!

Reply:
Thanks for catching and fixing this!

I feel it's probably better to add these in the plugin definition file (`src/plugin/index.ts`).  @kristw what do you think?

Reply:
umm, the regression is in `incubator-superset`, right?
Better call this in the setup stage of `incubator-superset`, not in the plugin. 

I configured it here for the storybook and haven't done that to the `incubator-superset` because we were still consolidating core packages at that time.
https://github.com/apache-superset/superset-ui/blob/0f11940d8201feea3c5f5adee82aef68a9d201f3/packages/superset-ui-demo/.storybook/preview.js#L70
See preview.
https://superset-ui.now.sh/?path=/story/chart-plugins-plugin-chart-word-cloud--legacy-shim

Reply:
or `src/plugin/index.ts` as Jesse suggested, but then have to do that for the other ones that use encodable too.

Reply:
This should only be required temporarily so either solution is fine. I am about to refactor the core package to use same underlying resolvers with encodable so this configuration should eventually be removed.

Reply:
Makes sense 👍  Updated.